content/en/docs/aws/deploy/install-kubeflow.md

```diff
@@ -216,7 +216,7 @@ kubectl rollout restart deployment dex -n auth
 
 Kubeflow provides multi-tenancy support and user are not able to create notebooks in `kubeflow`, `default` namespace.
 
-The first time you visit the cluster, you can ceate a namespace `anonymous` to use. If you want to create different users, you can create `Profile` and then `kubectl apply -f profile.yaml`. Profile controller will create new namespace and service account which is allowed to create notebook in that namespace.
+The first time you visit the cluster, you can create a namespace `anonymous` to use. If you want to create different users, you can create `Profile` and then `kubectl apply -f profile.yaml`. Profile controller will create new namespace and service account which is allowed to create notebook in that namespace.
```

Comment:
Good catch!

_Suggestion_

To avoid repetition, etc

```suggestion
The first time you visit the cluster, you can create a namespace `anonymous`. If you want to set up different users, you can create a `Profile` for each user and then run the following command: `kubectl apply -f profile.yaml`. A Profile controller will create a new namespace and a new service account for each new user, which allows to create notebook in that namespace.
```

@PatrickXYS @Jeffwan can you double check this please